frontend/src/components/filter/FilterGoalStandard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import FilterSelect from './FilterSelect';
+import { filterSelectProps } from './props';
+import useGoalTemplates from '../../hooks/useGoalTemplates';
+
+export default function FilterGoalStandard({
+  onApply,
+  inputId,
+  query,
+}) {
+  const goalTemplates = useGoalTemplates([]);
+
+  const standards = (goalTemplates || []).map((template) => ({
```

Comment:
Recommend `filterOptions` instead of `standards`, standards reads a little vague to me.